.lintstagedrc

```diff
@@ -1,7 +1,12 @@
 {
   "$schema": "http://json.schemastore.org/lintstagedrc.schema",
   "**/*.{js, ts}": [
```

Comment:
All of the `**/` prefixes in configurations are redundant AFAIK.

Reply:
`*.{}` will match all sub-folder?
I see this pattern in [eslint-config-create-app](https://github.com/facebook/create-react-app/blob/master/packages/eslint-config-react-app/index.js#L57)

Reply:
You should try yourself rather than be superstitious for authority.